elliptic-curve/src/public_key.rs

```diff
@@ -483,7 +506,11 @@ where
     where
         S: ser::Serializer,
     {
-        let der = self.to_public_key_der().map_err(ser::Error::custom)?;
+        // TODO: can we determine DER encoding length up-front? Using `MockCurve` gives
```

Comment:
Is there a way to collect this information from some associated const? Was hard to figure out